charts/dashboard/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: dashboard
+description: A dashboard for Diamond workflows
+type: application
+
+version: 0.1.0
+appVersion: 0.1.0-rc1
```

Comment:
Probably want to use the most recent tag on the repo here

Reply:
Yeah, saw that one myself and already changed it to rc-23